bucket/archwsl.json

```diff
@@ -43,30 +43,19 @@
         "    $installable = $false",
         "}",
         "if ($installable) {",
-        "    \"`$si = New-Object System.Diagnostics.ProcessStartInfo '$dir\\Arch.exe' -Property @{ UseShellExecute = `$false; RedirectStandardInput = `$true }",
-        "    `$p = [System.Diagnostics.Process]::Start(`$si)",
-        "    `$p.StandardInput.Write('y')",
-        "    `$p.WaitForExit()",
-        "    `$lx_uid = & '$dir\\Arch.exe' get --lxuid",
-        "    if(!`$lx_uid) { error 'ArchWsl is not installed!'; return }",
-        "    Set-ItemProperty -Path ('HKCU:Software\\Microsoft\\Windows\\CurrentVersion\\Lxss\\' + `$lx_uid) -Name BasePath -Value '$persist_dir\\data'",
-        "    if (Test-Path ('$persist_dir' + '\\data\\rootfs')) {",
-        "        Remove-Item -Recurse -Path '$dir\\rootfs'",
-        "    } else {",
-        "        Move-Item -Path '$dir\\rootfs' -Destination '$persist_dir\\data\\rootfs'",
-        "    }",
-        "    Remove-Item -Recurse -Path '$dir\\temp'",
+        "    \"Copy-Item '$dir\\Arch.exe' '$persist_dir\\data\\Arch.exe'",
```

Comment:
This will not work because `$dir` and `$persist_dir` can't be resolved when `'` is used instead of `"`

Reply:
Ok,changed in 91cfd36a
Thanks.